packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleTableColumnAnchorTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+#nullable enable
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleTableColumnAnchor"/>
+/// </summary>
+public class NimbleTableColumnAnchorTests
```

Comment:
Can't inherit from `NimbleAnchorBaseTests` because our column/wrapper doesn't define `href` (and doesn't derive from `NimbleAnchorBase`)